java/pom.xml

```diff
@@ -198,8 +198,30 @@
                         <configuration>
                             <excludes>
                                 <exclude>**/CuFileTest.java</exclude>
+                                <exclude>**/CudaTest.java</exclude>
                             </excludes>
                         </configuration>
```

Comment:
This should be moved into the first execution, as we don't want this configuration to apply to all executions.

Reply:
It seems that exclude only works at the plugin level. I don't know why, but it doesn't work if I move it into a specific execution. So, I use include to override the global exclude for the fatal test.